mgmt/ProxyConfig.h

```diff
@@ -98,15 +98,31 @@ class ConfigProcessor
 
 // A Continuation wrapper that calls the static reconfigure() method of the given class.
 template <typename UpdateClass> struct ConfigUpdateContinuation : public Continuation {
+  void (*f)();
   int
   update(int /* etype */, void * /* data */)
   {
     UpdateClass::reconfigure();
     delete this;
     return EVENT_DONE;
   }
+  int
+  update_func(int /*etype*/, void * /* data */)
```

Comment:
This code is useful when you have a ticket key file where content changes but you don't want to change the name of this file in records.config to reload it. Reconfigure() in general loads the whole config. So in a production environment, reloading the whole stack of certificates specified in ssl_multicert.config just for a ticket key update seems unnecessary. However, if there is any change in recods.config or ssl_multicert.config then the corresponding configparams get reloaded again, so that behavior has not changed. This ticket key is only required by the session ticket key callback, hence other config variables in sslconfigparams are not really dependent on this